Assets/JavaScript/index.js

```diff
@@ -0,0 +1,53 @@
+import Nav from '../../modules/Nav.js';
+import Books from '../../modules/Books.js';
+import { DateTime } from '../../node_modules/luxon/src/luxon.js';
+
```

Comment:
- [ ] Kindly ensure that the Luxon package is added as a .js file to the project (in full or minified version) and the module is imported e.g `import { DateTime } from "luxon";`